packages/itwinui-react/src/core/Popover/Popover.tsx

```diff
@@ -101,6 +101,10 @@ type PopoverInternalProps = {
    */
   trigger?: Partial<Record<'hover' | 'click' | 'focus', boolean>>;
   role?: 'dialog' | 'menu' | 'listbox';
+  /**
+   * Whether the popover should match the width of the trigger.
+   */
+  matchWidth?: boolean;
```

Comment:
1. I liked that we are giving the users this feature. I think it would be useful for them.
2. If we want to make it private, prop imports in components that use Popover needs to be updated as it is using only public props.

Reply:
i can't think of any use case where the user might want `matchWidth`, it's mostly useful for menus (which we already provide multiple components for). do you have another example in mind? if a user asks for it, we can always add it back in the future.

i've fixed the props types in [`d146a85`](https://github.com/iTwin/iTwinUI/pull/1598/commits/d146a859002a0e53538c30395b813a2ef02eee71)